src/CMakeLists.txt

```diff
@@ -221,6 +225,15 @@ function(generate_exports_file inputFilename versionName outputFilename)
                               PROPERTIES GENERATED TRUE)
 endfunction()
 
+macro (fetch_google_test)
+    include(FetchContent)
+    FetchContent_Declare(
+        googletest
+        URL ${GOOGLE_TEST_URL}
```

Comment:
So where does this actually pull from?

Reply:
See build/components/googletest/cgmanifest.json and src/build.sh. Build.sh reads the cgmanifest.json file to find the url snd sets this variable.

We need the cgmanifest file because our OSS component detection tools can't detect components downloaded by cmake.